osaasclient.go

```diff
@@ -163,3 +162,15 @@ func GetLogsForInstance(context *Context, serviceId, name, token string) ([]stri
 
 	return logs, nil
 }
+
+func AddServiceSecret(ctx *Context, serviceId, secretName, secretData string) error {
+	secretUrl := fmt.Sprintf("https://deploy.svc.%s.osaas.io/mysecrets/eyevinn-docker-retransfer", ctx.GetEnvironment())
```

Comment:
serviceId should not be hardcoded
```suggestion
	secretUrl := fmt.Sprintf("https://deploy.svc.%s.osaas.io/mysecrets/%s", ctx.GetEnvironment(), serviceId)
```